aspnet/fundamentals/localization/sample/Startup.cs

```diff
@@ -0,0 +1,175 @@
+﻿#define S
```

Comment:
@ryanbrandenburg 
I got rid of the ``#ifdef `` - It won't be there in my next update. I was experimenting with moving configuration code from ConfigureServices to Configure.